webapp/src/main/resources/META-INF/beans.xml

```diff
@@ -3,6 +3,8 @@
   xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd" version="1.1"
   bean-discovery-mode="all">
   <alternatives>
+    <class>org.trellisldp.webac.WebACService</class>
```

Comment:
You did this to "override" the presence of a no-op impl from `trellis-api`?

Reply:
In the Alternatives world (multiverse?), where implementations are in separate jars, you need to specify which alternative implementations are active in the beans.xml file.

Reply:
Right, that's what I meant. That's a chief reason I'm now worried about the no-op object pattern.

Reply:
Yep, it complicates the build modularity.

Reply:
For the nonce, we could get away with making the no ops `@Alternative`s and making all the real impls `@Default`s, eh? But we want something more elegant in the end. (Mumble mumble bean autodiscovery)

Reply:
I think I discovered the Defaults didn't work in a multi-jar setup IIRC, but I tried a lot of things.

Reply:
Fair enough. I think we have to bit the bullet and fix the no op pattern.